android/engine/src/main/assets/prameters_template.json

```diff
@@ -0,0 +1,23 @@
+{
+  "resourceType": "Parameters",
+  "parameter": [
+    {
+      "name": "practitioner",
+      "valueString": "#idPractitioner",
+      "part": [
+        {
+          "name": "CarePlan",
+          "valueString": "#idPartsCP"
+        },
+        {
+          "name": "Organization",
+          "valueString": "#idPartsO"
+        },
+        {
+          "name": "Location",
+          "valueString": "#idPartsL"
+        }
+      ]
+    }
+  ]
+}
```

Comment:
@FikriMilano Can you add this sample parameter here https://github.com/opensrp/fhir-resources/tree/main/general

Reply:
@dubdabasoduba 
This is actually no longer used, to be removed
Refer to our discussion https://github.com/opensrp/fhircore/issues/857#issuecomment-1207298098